pydm/widgets/image.py

```diff
@@ -153,23 +154,35 @@ def image_value_changed(self, new_image):
         """
         if new_image is None:
             return
-        if len(new_image.shape) == 1:
-            if self.image_width == 0:
-                self.image_waveform = new_image
-                self._needs_reshape = True
-                # We'll wait to draw the image until we get the width.
-                return
-            self.image_waveform = new_image.reshape((int(self.image_width), -1), order='F')
         elif len(new_image.shape) == 2:
             self.image_waveform = new_image
-        self.data_max_int = np.amax(self.image_waveform)  # take the max value of the recieved image
-        self.setColorMap()  # to update the colormap immediately
+        elif self._width_from_image_channel:
+            self.imageWidth = new_image[0]
```

Comment:
@fernandohds564 is this required by some special IOC that you guys have? Which EPICS Module is producing this waveform where the first element is the width?

Reply:
@hhslepicka a while ago we talked to the guy responsible for developing the IOC which will provide beam images from the Slits in our transport lines and he said that possibly the Area Detector they were considering had this behavior. That's why we have implemented this option.
However I just talked to him and he said that there will be separate channels for the width of the array, so it seems this option is not really needed. If you think this option is not necessary I can remove it.

Reply:
@fernandohds564 yes, please! Keep the idea of the width property but remove this concept of `width_from_image_channel`.

Reply:
ok @hhslepicka !

Reply:
It is done, @hhslepicka !